### PR TITLE
remove groupm bidder

### PR DESCRIPTION
**Pull request by pm-nilesh-chate**
Since PBS now has Marketplace support, there is no need for us to continue maintaining the GroupM bidder as the PubMatic Alias in PBS. Therefore, we are removing the bidder. This has already been taken care of in PBJS.

This resolves GroupM's issue at https://github.com/prebid/prebid-server/issues/2654

### CODE REVIEW

**Comment by Sonali-More-Xandr** [comment]
Can you please also reomve groupm [dev-docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/groupm.md)

**Comment by pm-nilesh-chate** [comment]
> Can you please also reomve groupm [dev-docs](https://github.com/prebid/prebid.github.io/blob/master/dev-docs/bidders/groupm.md)

https://github.com/prebid/prebid.github.io/pull/4598

File: exchange/bidder_test.go [hunk 1]
```diff
@@ -2725,7 +2725,7 @@ func TestExtraBidWithBidAdjustments(t *testing.T) {
 	}
 	bidAdjustments := map[string]float64{
 		string(openrtb_ext.BidderPubmatic): 2,
-		string(openrtb_ext.BidderGroupm):   3,
+		"groupm":                           3,
```
Comment (Sonali-More-Xandr) [inline exchange/bidder_test.go:2728]: Is there any reason why these references can't be removed completely?

**Comment by SyntaxNode** [comment]
Please update `GetDisabledBiddersErrorMessages` to include GroupM to avoid validation failures of older requests. There is no need to update unit tests for that change.

**Review by Sonali-More-Xandr — APPROVED** [review]

**Pull request merged**